silo-core/test/foundry/gas/Deposit1st.gas.sol

```diff
@@ -24,7 +24,7 @@ contract Deposit1stGasTest is Gas, Test {
             address(silo0),
             abi.encodeCall(ISilo.deposit, (ASSETS, BORROWER, ISilo.CollateralType.Collateral)),
             "Deposit1st ever",
-            168021
+            168127
```

Comment:
so more gas? :))

Reply:
This is the only change in gas tests.

Reply:
Is it not a first call tho? For the first call it makes sense because now it goes over 2 IFs. For each next call it will be cheaper because it will early exit skipping one IF.